tools/ansible/tasks/logging/test/logging_test.go

```diff
@@ -25,7 +33,17 @@ var requiredPluginList = map[string]string{
 	"fluent-plugin-prometheus":                 "",
 }
 
-// TestPluginsInstalled tests that the fluentd plugins were all installed correctly.
+// TestRubyInstalled tests if Ruby is installed correctly
+// This test should be run on a node after the logging playbook has been run against it
+// exec.Command requires fully qualified path for ruby.exe
+func TestRubyInstalled(t *testing.T) {
+	out, err := exec.Command(rubyExecutablePath, "--version").Output()
+	require.NoError(t, err)
+	actualVersion := strings.Split(string(out), " ")[1][:5]
```

Comment:
Example format again.

Reply:
I'll add the example in a comment here

Reply:
I think what @ravisantoshgudimetla was asking was for an example of the output `ruby --version`. Including that will help us understand how this works.